src/Database/Schema/PostgresSchema.php

```diff
@@ -409,17 +409,22 @@ public function columnSql(Table $table, $name)
 
         if (isset($data['null']) && $data['null'] === false) {
             $out .= ' NOT NULL';
+        } elseif (isset($data['null']) && $data['null'] === true) {
+            $out .= ' NULL';
```

Comment:
Is the inclusion of this `NULL` required. Aren't columns that are missing `NOT NULL` implicitly nullable?